lib/launchers/armrdu2.rb

```diff
@@ -0,0 +1,19 @@
+lib_path = File.expand_path(File.dirname(File.dirname(__FILE__)))
+unless $LOAD_PATH.any? {|p| File.expand_path(p) == lib_path}
+  $LOAD_PATH.unshift(lib_path)
+end
+
+require 'collections'
+require 'common'
+
+module BushSlicer
+  class ArmRdu2
```

Comment:
@aleskandro I suggest we follow the current class naming scheme (please refer to [OCMCluster class](https://github.com/openshift/verification-tests/blob/master/lib/launchers/o_c_m_cluster.rb#L17) for reference), so this should become `ARMRdu2` and the file name becomes `a_r_m_rdu2.rb`

Reply:
Done @pruan-rht. thanks.